blk/components/virt.c

```diff
@@ -29,10 +29,17 @@
 
 #define REQBK_SIZE BLK_QUEUE_SIZE_DRIV
 
+#ifndef BLK_VIRT_PASS_OFFSETS
```

Comment:
Now that I see the PR, I think we should actually remove this option and instead just fix the root cause with the driver VM so that it works with the virtualiser's address translation (my bad, I shouldn't have told you to introduce this config option).